dhis-2/dhis-api/src/main/java/org/hisp/dhis/programstagefilter/EventDataFilter.java

```diff
@@ -65,6 +65,12 @@ public class EventDataFilter implements Serializable {
   /** Like */
   private String like;
 
+  /** Null */
+  private String nullFilter;
+
+  /** Not Null */
+  private String notNullFilter;
```

Comment:
Maybe you already discussed this. But can't we simply reuse the nullFilter for both null and non null unary operator? FE anyway does a transformation.

Reply:
If we leave it up to FE to figure it out, then yes, we can use just one field for both operators.